### PR TITLE
Update MacOS and Linux with 2.1.0 update instructions

### PR DESCRIPTION
**Pull request by BClark09**
To be merged on next release.

Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

File: installation/linux.md [hunk 1]
```diff
@@ -607,13 +597,11 @@ sudo systemctl stop openhab2.service
...
-cd /tmp
-wget https://openhab.ci.cloudbees.com/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab/target/openhab-2.0.0-SNAPSHOT.zip
-sudo unzip openhab-2.0.0-SNAPSHOT.zip -d /opt/openhab2
-rm openhab-2.0.0-SNAPSHOT.zip
+You may restore these files by moving them back into your openhab folder:
 
+```shell
 # restore configuration and userdata
 sudo cp -arv /opt/openhab2-backup-$TIMESTAMP/conf /opt/openhab2/
 sudo cp -arv /opt/openhab2-backup-$TIMESTAMP/userdata /opt/openhab2/
```
Comment (ThomDietrich) [inline installation/linux.md]: Would you like to use a variable for the destination as done here: http://docs.openhab.org/installation/linux.html#backup-and-restore ??

**Comment by martinvw** [comment]
I did that indeed but it was not yet that high on my todo list. I'm a little sick now and just took a massive pain killer so time to go to sleep. Depending on my state tomorrow I will try it out.

File: installation/linux.md [hunk 2]
```diff
@@ -607,13 +597,11 @@ sudo systemctl stop openhab2.service
 # backup current installation with settings
 TIMESTAMP=`date +%Y%m%d_%H%M%S`;
```
Comment (BClark09) [inline installation/linux.md]: Sure!

File: installation/linux.md (hunk 1 above)
Comment (BClark09) [inline installation/linux.md]: I'll leave it with an example as is done in that section, as you wouldn't set a new variable for the restoration.

**Comment by BClark09** [comment]
> I did not read through the Mac part. Let's ping @martinvw for that, he "claimed" this area :)

Oops, I came in to add the manual installation parts and read this:

>`<!--Note to authors: If you revise this article, please consider adapting the Markdown style-->`

Hope you feel better @martinvw!

File: installation/linux.md [hunk 3]
```diff
@@ -625,6 +613,42 @@ sudo chown -hR openhab:openhab /opt/openhab2
...
+  ```
+
+* **Older Versions**
+
+  If you're using a version earlier than 2.1.0, then you can use the following commands in Terminal to download the script and run it. 
+  Assuming the openHAB directory is in `/opt/openhab`:
+
+  ```bash
+  cd /opt/openhab
+  sudo sh -c "$(curl -fsSL https://raw.githubusercontent.com/openhab/openhab-distro/master/distributions/openhab/src/main/resources/bin/update)"
```
Comment (martinvw) [inline installation/linux.md]: Is there a way to pass in a version number here? I tried but failed

File: installation/macosx.md [hunk 4]
```diff
@@ -5,69 +5,76 @@ title: Mac OS X
...
-* Move inside the `openhab` folder using the "cd" command.
+* **Snapshot (Unstable)**
+  
+  Snapshots are built daily, and contain the latest changes to openHAB's source code. 
+  These changes haven't undergone the same amount of testing as the stable releases, so only use them if you **need** a new feature.
+  Download the latest `openhab-2.x.x-SNAPSHOT.zip` [here](https://openhab.ci.cloudbees.com/job/openHAB-Distribution).
+  
+The following instructions are performed in a terminal window. At first launch of the terminal, you will be inside your home directory.
+
+Choose a folder name to host your install. Be aware that the path to this folder **must not contain any spaces**.
```
Comment (martinvw) [inline installation/macosx.md]: Is this also a problem on Mac I know it makes it harder

File: installation/linux.md (hunk 3 above)
Comment (BClark09) [inline installation/linux.md]: -c follows a slightly stranger syntax:

```bash
sh -c [command] -- [param 1] [param 2]
```

```bash
sudo sh -c "$(curl -fsSL https://raw.githubusercontent.com/openhab/openhab-distro/master/distributions/openhab/src/main/resources/bin/update)" -- 2.1.0
```

Do you think that we should put this specifically?

File: installation/macosx.md (hunk 4 above)
Comment (BClark09) [inline installation/macosx.md]: AFAIK, this is a problem with Karaf, which doesn't differentiate between OS for paths.

File: installation/linux.md (hunk 3 above)
Comment (BClark09) [inline installation/linux.md]: I thought about this, and being specific would be safer. WDYT?

Comment (martinvw) [inline installation/linux.md]: Yes, given this paragraph is for 2.0 to 2.1 this makes the most sense, thanks!

File: installation/macosx.md (hunk 4 above)
Comment (martinvw) [inline installation/macosx.md]: Ok, fine then!

**Comment by martinvw** [comment]
LGTM

**Comment by BClark09** [comment]
Thanks @martinvw!

**Comment by BClark09** [comment]
@ThomDietrich, @kaikreuzer and @ghys,

I've updated the windows instructions to be a little more in line in what should be happening in the future update script.

**Comment by ThomDietrich** [comment]
Excellent @BClark09! You are on fire the last couple of days :muscle:

**Comment by AngelosF** [comment]
I think that the http://docs.openhab.org/installation/linux.html#manual-installation steps need a small update for 2.1.0 on the download (wget) links also :)

**Pull request merged**